src/style.scss

```diff
@@ -40,11 +40,12 @@
 }
 
 :host {
-  height: 2.75rem;
+  line-height: 1;
+  --_labelpillpadding: .3125rem;
```

Comment:
Is this intentional? I don't know why the selector has `_` before the name? That is not a convention we use?

Reply:
The intent was to show that was a local variable vs a design token.  Do we have an existing pattern for that situation or should be just remove the underscore?

Reply:
We don't have a pattern for that since all design tokens start with `auro`. So any custom CSS property without `auro` is coming from somewhere other than design tokens.

Reply:
I am in favor of shipping this as is with the caveat that we create a new work item to figure out a pattern for local variables in the future.